tests/server/configs/sjrk.storyTelling.server.testServerWithBaseTheme.json5

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "sjrk.storyTelling.server.testServerWithCustomTheme",
+    "options": {
+        "gradeNames": ["sjrk.storyTelling.server"],
+        "components": {
+            "server": {
+                "options": {
+                    "globalConfig": {
+                        "binaryUploadDirectory": "./tests/server/uploads",
```

Comment:
Apparently tests don't make use of config values at line 9-10. Otherwise they won't pass with these configs being moved to `secureConfig`.

It would be nice to have integration tests to cover the communication between server and client so these configs make sense in tests.